hotline/templates/notifications/list.html

```diff
@@ -0,0 +1,53 @@
+{% extends "base.html" %}
+{% load notifications %}
+{% block content %}
+<div class="row">
+    <div class="col-md-12">
+        <h1>Searches you have subscribed to</h1>
+        <p>Here is a list of searches you have subscribed to. Once a search is deleted, it cannot be recovered. However, deleting a seach will not modify any information within the search.</p>
+    </div>
+</div>
+<div class="row">
+    <div class="col-md-12">
+        {% if form.errors %}
+        <div class="alert alert-warning">{{ form.errors }}</div>
+        {% endif %}
+        <form method="post">
+            <table class="table table-striped">
+                {% csrf_token %}
```

Comment:
Throw the `{% csrf_token %}` right after the `<form>`. Inside the `<table>` like that, it generates invalid HTML.